api/service/v1alpha1/service.proto

```diff
@@ -579,3 +583,11 @@ message DeleteAnalysisTemplateRequest {
 message DeleteAnalysisTemplateResponse {
   /* explicitly empty */
 }
+
+message ListProjectEventsRequest {
+  string project = 1;
```

Comment:
nit: We may need to introduce pagination in the future since we emit events on every Kargo events 👀

Reply:
IMO we should delay pagination for now (esp. given the 0.6 deadline looming). Events are ephemeral anyways so it's unlikely there will be overload.